include/mgp.hpp

```diff
@@ -2917,6 +2882,7 @@ inline bool Duration::operator<(const Duration &other) const {
 /* #region Value */
 
 inline Value::Value(mgp_value *ptr) : ptr_(mgp::value_copy(ptr, memory)) {}
+inline Value::Value(mgp_value **ptr) : ptr_(*ptr) {}
```

Comment:
I think you don't need this.

Reply:
Reason I added this part was because function Value::Value(mgp_value *ptr) : ptr_(mgp::value_copy(ptr, memory)) {} makes copy of mgp_value. In case of mgp::vertex_get_property, it returns copy of value. This is description of function:
"/// Get a copy of a vertex property mapped to a given name. Resulting value must be freed with mgp_value_destroy."

In case I make another copy, there will be one copy which is not deleted. And in that case there was memory leak. So with this function I am making a difference between case when we want copy, and when we don't want one.

Reply:
Ah I see it. Then the overloaded constructor is fine, however it is not clear for the interface. I would rather use something like:
```
struct StealType {}:

inline constexpr StealType steal{};

constexpr auto 

Value(Steal /*steal*/, mgp_value* ptr) : ptr_{ptr}{};
```

If you consider how do you call them:
```
mgp_value *ptr = ...;

auto value1 = Value(&ptr);
auto value2 = Value(steal, ptr);
```
I think `value2` is more clear. For more info/example, please checkout `std::pair`'s [ctor](https://en.cppreference.com/w/cpp/utility/pair/pair) (8th form) uses [`std::piecewise_construct_t`](https://en.cppreference.com/w/cpp/utility/piecewise_construct).

Reply:
Wow, really cool! Will add this. Thanks for the suggestion :smile:

Reply:
Check if it is okay now :)